lib/webpacker/commands.rb

```diff
@@ -5,6 +5,23 @@ def initialize(webpacker)
     @webpacker = webpacker
   end
 
+  def clean(count_to_keep = 2)
+    return if !config.public_output_path.exist? || !config.public_manifest_path.exist?
```

Comment:
@ericboehs I am wondering if we do this other way around for readability. 

```rb
if config.public_output_path.exist? && config.public_manifest_path.exist?
  # do the cleaning
end
```

Reply:
RuboCop usually barks at wrapping a whole method in a conditional and recommends using a guard clause such as this. I thought the way I wrote it was clear enough (I avoided using `unless`). Would it be easier to understand if I expanded it to two guard clauses?

```ruby
return unless config.public_output_path.exist?
return unless config.public_manifest_path.exist?

# ...
```

Reply:
Why not `return unless config.public_output_path.exist? && config.public_manifest_path.exist?`. Reads clearly to me.

Reply:
I usually stay away from compound `unless` statements but if I'm the oddball on thinking those are confusing I'd be glad to change it to your suggestion @swrobel.

Reply:
Thanks @ericboehs and @swrobel 

The reason mainly is that we don't use/prefer early returns inside Webpacker or Rails codebase. Rubocop shouldn't complain if it's multi-line conditional right?

Reply:
Okay. I updated it. I didn't realize the Rails style guide differed from the Ruby style guide here.